src/ballet/sbpf/fd_sbpf_loader.h

```diff
@@ -285,11 +310,12 @@ fd_sbpf_program_delete( fd_sbpf_program_t * program );
    Always returns a valid cstr, though the content is undefined in case
    the last call to `fd_sbpf_program_load` returned zero (success). */
 
-char const *
-fd_sbpf_strerror( void );
-
 /* SIMD-0189 */
 static inline int fd_sbpf_enable_stricter_elf_headers( ulong sbpf_version ) { return sbpf_version >= FD_SBPF_V3; }
+/* SIMD-0178 */
+static inline int fd_sbpf_static_syscalls            ( ulong sbpf_version ) { return sbpf_version >= FD_SBPF_V3; }
```

Comment:
I moved `fd_sbpf_enable_stricter_elf_headers` here from `fd_vm_private.h` (and change it from macro to static inline as per richie recommendation).

Maybe we should unify/avoid duplicated definitions, for example now we have both `FD_VM_SBPF_STATIC_SYSCALLS` and `fd_sbpf_static_syscalls`.